1-mandatory/2-quotegenerator/quotes.js

```diff
@@ -1,3 +1,12 @@
+const getNewQuote = async () =>
+{const indx = Math.floor(Math.random()*quotes.length);
+    document.getElementById("quoteDisplay").innerHTML=quotes[indx].quote;
+    document.getElementById("authorDisplay").innerHTML="~ "+quotes[indx].author;
+}
+getNewQuote();
+
+
```

Comment:
Good catch! so you understand now how does this expression works? 
```js
Math.floor(Math.random() * quotes.length);
```
for this task `getNewQuote()` could just a regular function, not the async one. coz you don't interract with any source of data outside of your own code. Array of quotes is inside your project and you load it into a variable.

other minor things to note is probably formatting and readability:
```js
const getNewQuote = () => {
//                       ^^^ better place for the opening bracket
};
```

```js
Math.floor(Math.random() * quotes.length);
                       ^^^^^ use spaces to separate different part of expression
```
same here:
```js
document.getElementById("authorDisplay").innerHTML="~ "+quotes[indx].author;

// it looks and reads better this way:
document.getElementById("authorDisplay").innerHTML = "~ " + quotes[indx].author;
```